doc/functions/fhs-environments.xml

```diff
@@ -50,6 +50,58 @@
     </para>
    </listitem>
   </varlistentry>
+  <varlistentry>
+    <term>
+      <literal>automount</literal>
```

Comment:
I don't think this should be an option, not for `buildFHSUserEnv`, that is: it should really be the only behavior, because isolation is not the goal (FHS compat is).

Later, we can get to sandboxing of individual derivations, XDG portals, etc. That said, it will certainly require a complete redesign of `buildFHSUserEnv` and will have to be carefully thought out.